react/Menu/index.jsx

```diff
@@ -97,13 +97,15 @@ class Menu extends Component {
       position,
       component,
       itemsStyle,
-      popover
+      popover,
+      ...restProps
     } = this.props
     const { opened } = this.state
     const Tag = popover === true ? Popover : 'div'
 
     return (
       <div
+        {...restProps}
```

Comment:
`{...restProps}` should be put at the end of the tag to be able to override any default behavior IMHO.